MinPlatformPkg/Include/Library/PlatformConfigCheckLib.h

```diff
@@ -0,0 +1,26 @@
+/** @file
+
+  Platform configuration check library for TestPointCheckLib
+
+  Copyright (c) Microsoft Corporation. All rights reserved
+
+**/
+
+#ifndef _PLATFORM_CONFIG_CHECK_LIB_H_
+#define _PLATFORM_CONFIG_CHECK_LIB_H_
+
+#include <Uefi.h>
+
+/**
+  This function dumps platform level information at Exit Boot Services.
+
+  @retval     EFI_SUCCESS  Function has completed successfully.
+              Other        Function error indicates failure.
+**/
+EFI_STATUS
+EFIAPI
+PlatformConfigDumpExitBootServices (
+  VOID
+  );
+
+#endif // _PLATFORM_CONFIG_CHECK_LIB_H_
```

Comment:
As a general comment, I recommend leaving a single empty line at the end of each file. This is more important in source files targeting open source which might end up being exposed to a wider set of tools that depend on this.

Reply:
There is an empty line after the #endif here, but for some reason it doesn't show up in the PR. My guess is the UI automatically trims it.